tests/Feature/Http/Controllers/Admin/SpaceControllerTest.php

```diff
@@ -47,7 +47,7 @@ public function testCreateFormIsShown(): void
     public function testThatANewSpaceCanBeCreated(): void
     {
         // Given
-        $user = User::factory()->create();
+        $user = AuthorizedUser::factory()->create();
```

Comment:
Indien mogelijk, de variabele naam aanpassen naar `$authorizedUser` .

```suggestion
        $authorizedUser = AuthorizedUser::factory()->create();
````